itou/utils/management/commands/s3_antivirus_scan_resume_files.py

```diff
@@ -0,0 +1,60 @@
+import os
+import time
+
+import pyclamd
+import requests
+from django.core.management.base import BaseCommand, CommandError
+from tqdm import tqdm
+
+from itou.job_applications.models import JobApplication
+
+
+class Command(BaseCommand):
+    help = "Run ClamAV antivirus scan on resume files hosting in S3 storage type"
+
+    def handle(self, *args, **options):
+        # check that clamav is deployed in the current environment
+        if os.environ.get("CC_CLAMAV", False):
+            infected_files = []
+            self.stdout.write("Scan starts..")
+            job_applications = JobApplication.objects.exclude(resume_link__isnull=True).exclude(resume_link__exact="")
+            with tqdm(total=job_applications.count()) as pbar:
+                for job_application in job_applications:
+                    file_url = job_application.resume_link
+                    filename = file_url.split("/")[-1]
+                    filepath = f"/tmp/{filename}"
+
+                    # Download the file
+                    start_counter = time.perf_counter()
+                    self.stdout.write(f"File download starts ({filename}) for JobApplication #{job_application.pk}")
+                    r = requests.get(file_url)
+                    with open(filepath, "wb") as download_file:
+                        download_file.write(r.content)
+                    self.stdout.write(f"File download completed in {time.perf_counter()-start_counter}s ({filename})")
+
+                    # Scan the file with ClamAV
+                    start_counter = time.perf_counter()
+                    self.stdout.write(f"Scanning file {filename}")
+                    try:
+                        cd = pyclamd.ClamdAgnostic()
+                        result = cd.scan_file(filepath)
```

Comment:
OMG pyclamd `This version is compatible with python 3 (tested with 3.4.3) and python 2 (tested 2.7.3).`

Apparemment c'est juste un fichier python tout bete, je me demande si on pourrait pas juste le copier et maintenir chez nous, ça serait peut etre plus facile à maintenir (dernière MAJ 2017)